README.md

```diff
@@ -380,7 +402,7 @@ method takes two arguments:
 ### Profunctor
 
 A value that implements the Profunctor specification must also implement
-the Functor specification.
+the Functor and Contravariant Functor specifications.
```

Comment:
I see! I think. Here's a statement my understanding would predict to be true: there's a `Contravariant` instance for `Op`, but no `Functor`. Is that correct?

I'm still not quite sure I get

> In some other typed languages we'd have an issue where the variance of the type arguments of contravariant and profunctors don't line up

In that I think I see the problem now, but I'm not sure I see it's relation to variance of the type arguments

Also there seems to me to be an inessential semantic issue at hand. In that one speaks of, say, the functor instance for functions, but it's seems somewhat conventional / tied to Haskell's syntax that we say fixing the first type variable gives us 'the' functor instance. Might it not be more accurate to talk of the functor instance for functions of fixed domain?

It may be that I have a fairly deep misunderstanding here that would take a lot of teaching effort to resolve, so will happily accept being told to go and read something etc.